app/src/main/java/com/jerboa/MainActivity.kt

```diff
@@ -135,6 +145,27 @@ class MainActivity : ComponentActivity() {
 
                 ShowChangelog(appSettingsViewModel = appSettingsViewModel)
 
+                when (val siteRes = siteViewModel.siteRes) {
```

Comment:
This needs to be externalized, and also added to the login page.

Reply:
This will show two alerts when first launching the app, this one just before the change log.

Would you prefer the change log first? Or have it be one or the other?